config/lib/app.js

```diff
@@ -11,6 +11,11 @@ var config = require('../config'),
 // Initialize Models
 mongoose.loadModels();
 
+// Populate DB with sample data
+if (config.seedDB) {
+	require('./seed');
```

Comment:
Have u seen [this](http://stackoverflow.com/questions/2316677/can-git-automatically-switch-between-spaces-and-tabs)?